### ISSUE TITLE
drill: reuse the TCP connection 

### ISSUE THREAD

**Issue by dilyanpalauzov** [NLnetLabs/ldns#57]
When I call `drill -DTtV5 TLSA _25._tcp.mail.aegee.org`, it prints some `SERVER: \<ip-address\>` sections and the IP address changes over time to the different name servers offered for “aegee.org”.

Does drill on purpose do not use reuse the TCP connection established to one of the name servers?

**Comment by wtoorop** [comment]
No, re-using open TCP connections has simply not (yet) been implemented
